src/chainstate/stacks/db/unconfirmed.rs

```diff
@@ -207,7 +207,7 @@ impl UnconfirmedState {
 
                 last_mblock = Some(mblock_header);
                 last_mblock_seq = seq;
-                new_bytes = {
+                new_bytes += {
```

Comment:
Should we add a regression test for this?

Reply:
Yes, we should.

Reply:
I think this test can actually be expanded.  The difficulty with testing microblock mining is that the current Neon node design makes it very difficult to construct invalid or malicious microblock streams; the bugs we've had with microblocks stem from the node confirming an invalid stream.

I think I've found a better way to test this through fault injection.  Going to spend cycles tomorrow doing this not only for size, but also compute budget.